packages/ui/src/components/Table.svelte

```diff
@@ -71,6 +72,19 @@
     list?.reset()
   }
   let checked: boolean = false
+
+  function asComponent (comp: UIComponent | AnyComponent): AnyComponent {
+    return comp as AnyComponent
+  }
+
+  function getValue (doc: any, key: string): any {
```

Comment:
Could you try to use '@anticrm/core' getNestedValue() method for same, it also could query for arrays.